catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -264,6 +264,16 @@
         <bundle>mvn:ddf.catalog.admin/catalog-admin-module-sources/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-transformer-streaming-api" install="manual" version="${project.version}"
+             description="DDF Generic Input Transformer API">
+             <bundle>mvn:ddf.catalog.transformer/catalog-transformer-streaming-api/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-transformer-streaming-impl" install="manual" version="${project.version}"
```

Comment:
Do these really need to be separate features? Unsure why you'd want one without the other.